pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[project]
+name = "remote-server"
+version = "0.1.0"
+description = "NVDA Remote Access remote relay server."
+readme = "README.md"
+requires-python = ">=3.13.3"
+dependencies = [
+    "pyopenssl>=25.1.0",
+    "service-identity>=24.2.0",
+    "twisted>=24.11.0",
+]
+
+[dependency-groups]
+dev = [
+    "pre-commit>=4.2.0",
+    "pyright>=1.1.401",
+    "ruff>=0.11.12",
+]
```

Comment:
these are safer to leave unpinned. It might a bit annoying though if ruff updates or pyright changes create inconsistent rule violations between environments. I would also consider pinning these to `~=X.X`